internal/terraform/node_resource_plan_instance.go

```diff
@@ -622,7 +623,7 @@ func (n *NodePlannableResourceInstance) planActionTriggers(ctx EvalContext, chan
 				if err != nil {
 					diags = diags.Append(tfdiags.Sourceless(
 						tfdiags.Error,
-						"Failed to get provider",
+						fmt.Sprintf("Failed to get provider for action %s", absActionAddr),
```

Comment:
same here, can you get the action source address and put it into subject instead of changing the summary?